### DISCUSSION TITLE
Task node style

### DISCUSSION THREAD

**Discussion by mbnoimi** [phase1geo/Minder#582] [Q&A]
How can I change the style of task node?
Current style is horrible (unreadable) yellow background for completed task!

![image](https://github.com/phase1geo/Minder/assets/284547/246e5c88-fcda-4e2b-bbcb-150c426b02ef)

**Comment by phase1geo** [comment]
This doesn't look right.  An incomplete task will have a border the color of the link and filled with the  background color.  A completed task will have the border the color of the background and filled with the link color.  The latest version of Minder is 1.15.6.

**Reply by mbnoimi** [reply]
>  The latest version of Minder is 1.15.6.

I'm using the same recent version (flatpak ver)!

**Reply by mbnoimi** [reply]
Example:
![image](https://github.com/phase1geo/Minder/assets/284547/5f1977d2-38f1-425d-bde0-1dc98323b1e1)

[tasks_style.minder.zip](https://github.com/phase1geo/Minder/files/12884376/tasks_style.minder.zip)

**Comment by phase1geo** [comment]
Well, this is really weird.  If I use the Minder flatpak (1.15.6) and open this file, I see the same thing that you do.  However, if I run Minder 1.15.6 from source, it looks correct.  The code that draws tasks follows what I would expect.  So I don't know why the Flatpak version is doing something different and against the codebase.  I'll try to keep looking into this, but the problem is that Flatpaks are hard to debug, so if I could recreate this issue from source, it would be easier to understand what is going on.
